Configurations/10-main.conf

```diff
@@ -1317,10 +1327,13 @@ my %targets = (
     "VC-common" => {
         inherit_from     => [ "BASE_Windows" ],
         template         => 1,
-        cc               => "cl",
-        cpp              => '$(CC) /EP /C',
-        cflags           => "-W3 -wd4090 -Gs0 -GF -Gy -nologo",
-        defines          => add("OPENSSL_SYS_WIN32", "WIN32_LEAN_AND_MEAN",
+        CC               => "cl",
+        CPP              => '$(CC) /EP /C',
+        CFLAGS           => "-W3 -wd4090 -Gs0 -GF -Gy -nologo",
```

Comment:
/GF is read-only string pooling. Kind of like -qroconst in aix-cc and there is non-overridable. One can probably say same about all /G options...

Reply:
Okie

Reply:
(although, as a side note, I was wondering of things like `-qroconst` should be possible to override.  Not that I can understand why anyone would, but...)

Reply:
Things like -qroconst is rather a declaration of code properties. This one means that there is no code in **our** code base that will try to write to a constant. And without this option you wouldn't be sure. It's also sensible from security perspective, because it kind of ensures that buggy application code can't tamper with our security-sensitive constants. In other words I'd say it appropriate to keep it non-overridable.

Reply:
On related note one can probably say following. Primary criterion for determining if option should be overridable or not is whether or not option in question **has to be** harmonized with option used to compile application code. Simplest example is debugging options. I mean if you want to debug application, you need debugging library build, right.

Reply:
> ... In other words I'd say it appropriate to keep it non-overridable.

Agreed.